Bottom: 973111bcba6d834e198e1d8e14afad203a8d2790
Top:    f80eb80e16768d162a49b644071e1047a9610190
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 23:06:11 +0100

Refresh of draw-c-change-drawtop-to-drawo

---

diff --git a/src/draw.c b/src/draw.c
index 5526a31..2ea2467 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -569,7 +569,7 @@ pad_callback (const BoxType * b, void *cl)
 }
 
 /* ---------------------------------------------------------------------------
- * draws pins pads and vias
+ * draws outer layer pins pads and vias
  */
 static void
 DrawOuterPPV (int side, const BoxType * screen)
